### PR TITLE
An9/wi00692027/ncn-Introduced Enabled property to handle the dragNewLinkBehaviour

### PR DESCRIPTION
**Pull request by AarthiNarayanan**
Introduced Enabled property to handle the dragNewLinkBehaviour

### CODE REVIEW

File: src/Blazor.Diagrams.Core/Behaviors/DragNewLinkBehavior.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ private void OnPointerDown(Model? model, MouseEventArgs e)
 
         if (model is PortModel port)
         {
-            if (port.Locked)
+            if (port.Locked || port.Enabled)
```
Comment (suraj-desai-wtg) [inline src/Blazor.Diagrams.Core/Behaviors/DragNewLinkBehavior.cs:59]: Shouldn't it be !port.Enabled?

Comment (AarthiNarayanan) [reply src/Blazor.Diagrams.Core/Behaviors/DragNewLinkBehavior.cs]: Yes, when I changed the name I forgot change the implementation as per the name, I have changed it now.

File: tests/Blazor.Diagrams.Core.Tests/Behaviors/DragNewLinkBehaviorTests.cs [hunk 2]
```diff
@@ -21,7 +21,7 @@ public void Behavior_ShouldCreateLinkWithSinglePortAnchorSource_WhenMouseDownOnP
         {
             Initialized = true,
             Position = new Point(110, 60),
-            Size = new Size(10, 20)
+            Size = new Size(10, 20),
```
Comment (suraj-desai-wtg) [inline tests/Blazor.Diagrams.Core.Tests/Behaviors/DragNewLinkBehaviorTests.cs:24]: Minor: We don't need this change anymore after changing to !Enabled

**Review by suraj-desai-wtg — CHANGES_REQUESTED** [review]
PR is ready to merge after addressing a minor comment that I left

File: tests/Blazor.Diagrams.Core.Tests/Behaviors/DragNewLinkBehaviorTests.cs (hunk 2 above)
Comment (AarthiNarayanan) [reply tests/Blazor.Diagrams.Core.Tests/Behaviors/DragNewLinkBehaviorTests.cs]: Removed comma

**Pull request merged**